src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -746,6 +797,7 @@ private static String executeRequestInternal(
               includeRequestGuid,
               retryOnHTTP403,
               execTimeData);
+       stopwatch.stop();
```

Comment:
do we want to measure time only when debug is enabled?

Reply:
Changed it to count time only if the debug is enabled.